assets/js/admin/reports/charts/utils.js

```diff
@@ -132,3 +133,31 @@ export const toolTipBaseConfig = {
 		tooltipEl.style.padding = tooltip.yPadding + 'px ' + tooltip.xPadding + 'px';
 	},
 };
+
+/**
+ * Attach formatting callback to axis ticks.
+ *
+ * @param {Object} ticks Axis ticks configuration.
+ */
+ export const attachAxisTickFormattingCallback = ( ticks ) => {
+	const number = new NumberFormat();
+
+	ticks.callback = function( value, index, values ) {
+		switch ( ticks.formattingType ) {
+			case 'raw':
```

Comment:
Should raw and the default be combined? I'm a little concerned that there isn't a default...although actually defining this is a little redundant so maybe we just need integer and format cases, and don't bother defining anything for raw/default?